agenta-web/src/components/pages/evaluations/evaluators/NewEvaluatorModal.tsx

```diff
@@ -266,34 +266,38 @@ const NewEvaluatorModal: React.FC<Props> = ({
 
     const advancedSettingsFields = evalFields.filter((field) => field.advanced)
     const basicSettingsFields = evalFields.filter((field) => !field.advanced)
-    console.log(evalFields)
-    console.log(basicSettingsFields)
 
     const onSubmit = (values: CreateEvaluationConfigData) => {
-        setSubmitLoading(true)
-        if (!selectedEval?.key) throw new Error("No selected key")
-        const settingsValues = values.settings_values || {}
+        try {
+            setSubmitLoading(true)
+            if (!selectedEval?.key) throw new Error("No selected key")
+            const settingsValues = values.settings_values || {}
 
-        if (
-            !settingsValues.correct_answer_key &&
```

Comment:
@bekossy I see that this has been added to my PR [here](https://github.com/Agenta-AI/agenta/pull/1711/commits/7bffe152b4eed9b8c01a32e501817046fdf39c0c). That is wrong. Why has this been added?